scripts/generate-dependencies/generate-dependencies.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Adds package dependencies to the ripple package.json file based on available
+ * packages.
+ *
+ * Requires each ripple package to have:
+ *
+ *   1. A `package.json` file with:
+ *     - name
+ *
+ * Script expects to be called via npm in the root directory:
+ *
+ *   `npm run package-dependencies`
+ */
+
+let fs = require('fs')
+let glob = require('glob-fs')({ gitignore: true })
+
+// Get Package JSON.
+let packageJson = fs.readFileSync('./package.json')
+packageJson = JSON.parse(packageJson)
+
+// Strip existing @dpc-sdp keys.
+for (let key in packageJson.dependencies) {
+  if (key.indexOf('@dpc-sdp') === 0) {
```

Comment:
May add `ripple` namespace `@dpc-sdp/ripple-` incase we have other dependencies under `@dpc-sdp` which is not part of Ripple.

Reply:
Updated as per comment. Added documentation to Contributing file.